lib/IDE/CursorInfo.cpp

```diff
@@ -411,7 +413,7 @@ class CursorInfoDoneParsingCallback : public IDEInspectionCallbacks {
       break;
     }
     if (Result) {
-      Consumer.handleResults(*CursorInfo);
+      Consumer.handleResults(CursorInfo);
     }
```

Comment:
`CursorInfo` could have been null here before 😅. I'm honestly not sure why it doesn't actually cause any issues though. It does on Linux with C++ stdlib assertions enabled. Anyway, looks like this fixes it (assuming `handleResults` handles `nullptr`).